internal/cryptoutil/cryptoutil.go

```diff
@@ -33,6 +38,12 @@ func CreateSigner(kms, name string, opts ...pemutil.Options) (crypto.Signer, err
 	return newKMSSigner(kms, name)
 }
 
+// CreateAttestor creates an attestor that will use `step-kms-plugin` with the
+// given kms and uri.
```

Comment:
It says `given kms and uri`, but it takes a kms and name.

Reply:
Fixed with 6ad024e